buildSrc/src/main/kotlin/android-base-lib.gradle.kts

```diff
@@ -48,8 +48,8 @@ dependencies {
 val androidSourcesJar by tasks.registering(Jar::class) {
     archiveClassifier.set("sources")
     if (project.plugins.findPlugin("com.android.library") != null) {
```

Comment:
@HectorNarvaez @hans-hamel What's the point to validate this here? This pre-compiled plugin applies the android library plugin at the beginning of the script so this will be always true, isn't it?